scripts/update_lib/tests/test_auto_mark.py

```
@@ -747,5 +781,156 @@ async def test_one(self):
+======================================================================
+FAIL: test_foo (test.test_example.TestA.test_foo)
+----------------------------------------------------------------------
+Traceback (most recent call last):
+  File "test.py", line 10, in test_foo
+    self.assertEqual(1, 2)
+AssertionError: 1 != 2
+"""
+
+    def test_auto_mark_file_crashed_run(self):
+        """auto_mark_file processes results even when tests_result is empty (crash)."""
+        test_code = f"""import unittest
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)